### PR TITLE
pyWhat 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by BastianZim**
pyWhat 1.1.0 (new formula)

Add formula

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/pywhat.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+class Pywhat < Formula
+  include Language::Python::Virtualenv
+
+  desc "What is that thing?"
```
Comment (carlocab) [inline Formula/pywhat.rb:4]: Can we use a slightly more informative description?

File: Formula/pywhat.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  resource "rich" do
+    url "https://files.pythonhosted.org/packages/1e/cc/ced09195051b5384e9a82d6de7fc1a3917017fe214d30d41a9935cea465d/rich-10.2.2.tar.gz"
+    sha256 "17b3f486c38e79cc219d8848974b277ef532a82d12b3ad6eb37bb8c6f22ab5fc"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/71/39/171f1c67cd00715f190ba0b100d606d440a28c93c7714febeca8b79af85e/six-1.16.0.tar.gz"
+    sha256 "1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"
+  end
```
Comment (carlocab) [inline Formula/pywhat.rb:51]: We can replace this with `depends_on "six"`.

**Review by carlocab — COMMENTED** [review]
Thanks for your PR, @BastianZim. Just a few suggestions.

**Comment by BastianZim** [comment]
Thanks for the review @carlocab!

File: Formula/pywhat.rb (hunk 1 above)
Comment (BastianZim) [reply Formula/pywhat.rb]: Original description from: https://pypi.org/project/pywhat/
New description from: https://github.com/bee-san/pyWhat

File: Formula/pywhat.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+class Pywhat < Formula
+  include Language::Python::Virtualenv
+
+  desc "🐸 Identify anything. pyWhat easily lets you identify emails, IP addresses, and more. Feed it a .pcap file or some text and it'll tell you what it is! 🧙‍♀️"
```
Comment (carlocab) [inline Formula/pywhat.rb:4]:
```suggestion
  desc "🐸 Identify anything: emails, IP addresses, and more 🧙‍♀️"
```

File: Formula/pywhat.rb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    url "https://files.pythonhosted.org/packages/ba/6e/7a7c13c21d8a4a7f82ccbfe257a045890d4dbf18c023f985f565f97393e3/Pygments-2.9.0.tar.gz"
+    sha256 "a18f47b506a429f6f4b9df81bb02beab9ca21d0a5fee38ed15aef65f0545519f"
+  end
+
+  resource "rich" do
+    url "https://files.pythonhosted.org/packages/1e/cc/ced09195051b5384e9a82d6de7fc1a3917017fe214d30d41a9935cea465d/rich-10.2.2.tar.gz"
+    sha256 "17b3f486c38e79cc219d8848974b277ef532a82d12b3ad6eb37bb8c6f22ab5fc"
+  end
+
+  depends_on "six"
```
Comment (carlocab) [inline Formula/pywhat.rb:48]: Move this to the line immediately after
```ruby
depends_on "python@3.9"
```

**Comment by BastianZim** [comment]
Was changed @carlocab

**Review by carlocab — APPROVED** [review]
Thanks, @BastianZim.

**Comment by BastianZim** [comment]
Thanks for the reviews @carlocab!

**Pull request closed without merging**